hal-core/src/mem/page.rs

```diff
@@ -2,13 +2,13 @@ use crate::{Address, PAddr, VAddr};
 use core::{cmp, fmt, ops, slice};
 
 pub trait Size: Copy + Eq + PartialEq + fmt::Display {
-    /// The size (in bits) of this page.
-    fn size(&self) -> usize;
+    /// Returns the size (in bytes) of this page.
+    fn in_bytes(&self) -> usize;
```

Comment:
I think it is more common to word this as `as_bytes` or `to_bytes`, given it takes `&self`, and returns another interpretation of the value.

That being said, I suppose that name might suggest that it's returning the size as `&[u8]`, which would be a bit weird. Not a huge deal either way.

Perhaps `as_usize` would be reasonable here (if less descriptive)?

Reply:
my thought process is that numbers that represent a value with a unit ought to be returned by methods with that unit in their names, and that, as you said, `as_bytes`/`to_bytes` really sound like conversions to a byte slice/array/whatever. the guidelines for conversions don't really deal with _unit_ conversions, only type conversions, so i picked `in_` kinda arbitrarily. but, i also liked that you get a nice little sentence --- `page.size().in_bytes()` and similar. 

`as_usize` seems like strictly less information, to me. but i'm open to change it if you don't like this.

Reply:
changed it to `as_usize`